src/System.Web.HttpUtility/src/System/Web/Util/HttpEncoder.cs

```diff
@@ -283,45 +283,14 @@ internal static byte[] UrlDecode(byte[] bytes, int offset, int count)
                 }
                 else if (b == '%' && i < count - 2)
                 {
-                    if (bytes[pos + 1] == 'u' && i < count - 5)
-                    {
-                        int h1 = HttpEncoderUtility.HexToInt((char)bytes[pos + 2]);
```

Comment:
> This file was a direct copy from .NET Framework

@alexperovich, do you know how this:
https://referencesource.microsoft.com/#System.Web/Util/HttpEncoder.cs,477
ended up differing from this:
https://github.com/dotnet/corefx/blob/3bbc5ddc7006bafa0d68883dde90fbd0f839d57c/src/System.Web.HttpUtility/src/System/Web/Util/HttpEncoder.cs#L284-L291
?